src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java

```diff
@@ -232,11 +232,7 @@ protected SandboxedSpawn prepareSpawn(Spawn spawn, SpawnExecutionContext context
 
     SandboxInputs inputs =
         helpers.processInputFiles(
-            context.getInputMapping(
-                getSandboxOptions().symlinkedSandboxExpandsTreeArtifactsInRunfilesTree),
-            spawn,
-            context.getArtifactExpander(),
-            execRoot);
+            context.getInputMapping(true), spawn, context.getArtifactExpander(), execRoot);
```

Comment:
Do we call `getInputMapping` with `false` anywhere?

Reply:
Not as far as I can see in Bazel. I was hoping to follow up with a CL removing that parameter if someone could assure me that Google is not passing `false`.

Reply:
Ok, let's do that in a separate CL then.

Reply:
https://bazel-review.googlesource.com/c/bazel/+/137717

Reply:
Thanks—I also raced and submitted #11809.

Reply:
Sorry for the duplicate work, I thought it's easier for me to do with the internal changes that were necessary.

Reply:
No problem; it was almost completely mechanical anyway.